docs/usage/config-validation.md

```diff
@@ -18,17 +18,29 @@ When you run `renovate-config-validator` with no arguments it will check:
 For example:
 
 ```console
-$ npm install --global renovate
-added 750 packages, and audited 751 packages in 51s
-$ renovate-config-validator
+$ npx --yes --package renovate -- renovate-config-validator
  INFO: Validating renovate.json
  INFO: Config validated successfully
 ```
 
 ### Strict mode
 
 By default, the validator program fails with a non-zero exit code if there are any validation warnings or errors.
-You can pass the `--strict` flag to make it fail if a scanned config needs migration.
+You can pass the `--strict` flag to make it fail if a scanned config needs migration:
+
+```console
```

Comment:
```suggestion
```console title="Strict mode leads to validation failure if config needs migration"
```

How about a nice "title" for the codeblock? See: [Material for MkDocs manual, adding a title to a codeblock](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title).